ncl/ncl_entries/climatology_functions.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>long-term with hyphen</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This (and the others) came from the NCL descriptions, but I'll go through and edit these.  

We probably don't need to propagate grammatical errors.